sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationManager.kt

```diff
@@ -155,6 +157,17 @@ interface ViewAnnotationManager {
    */
   fun getViewAnnotationUpdateMode(): ViewAnnotationUpdateMode
 
+  /**
+   * Sets the visible region of the map that displays all the given views.
+   * This API is not supported by Globe projection.
```

Comment:
looking at the impl now I don't see that use-case handled; can we actually check if globe projection is enabled and throw some our Mapbox exception then?

Reply:
I took the docs from `cameraForCoordinateBounds` which doesn't support Globe projection. 

> can we actually check if globe projection is enabled and throw some our Mapbox exception then?

Yeah good idea,  i will add exception when using globe.